GOVERNANCE.md

```diff
@@ -0,0 +1,235 @@
+Examples:
+| Votes Cast | Yes Votes | No Votes | Required Yes Votes (≥2/3) | Result               |
+|------------|-----------|----------|----------------------------|----------------------|
+| 9          | 6         | 3        | 6                          | ✅ Passes (6 = 2/3)   |
+| 9          | 5         | 4        | 6                          | ❌ Fails (5 < 2/3)    |
+| 6          | 4         | 2        | 4                          | ✅ Passes (4 = 2/3)   |
+| 6          | 3         | 3        | 4                          | ❌ Fails (3 < 2/3)    |
+| 12         | 8         | 4        | 8                          | ✅ Passes (8 = 2/3)   |
+| 12         | 7         | 5        | 8                          | ❌ Fails (7 < 2/3)    |
+ 
+
+## Simple majority
+
+A [**Simple majority**](#simple-majority) is defined as: more than half of the votes cast in a decision-making process.
+
+Examples:
+| Votes Cast | Yes Votes | No Votes | Result               |
+|------------|-----------|----------|----------------------|
+| 10         | 6         | 4        | ✅ Passes (6 > 5)     |
+| 10         | 5         | 5        | ❌ Fails (not > 50%)  |
+| 7          | 4         | 3        | ✅ Passes (4 > 3)     |
+| 7          | 3         | 3        | ❌ Fails (tie; not >) |
+
+
+## Voting and Decision Making
+
+While most business in K3s is conducted by "[lazy consensus](https://community.apache.org/committers/lazyConsensus.html)", periodically the Maintainers may need to vote on specific actions or changes. A vote can be taken on [the developer mailing list](mailto:cncf-k3s-dev@lists.cncf.io) or [the private Maintainer mailing list](mailto:cncf-k3s-maintainers@lists.cncf.io) for security or conduct matters.  
+Votes may also be taken at [the community meeting](https://k3s.io/community/#community-meetings).  Any Maintainer may demand a vote be taken.
+
+Most votes require a [simple majority](#simple-majority) of all Maintainers to succeed, except where otherwise noted.  [Supermajority](#Supermajority) votes mean at least two-thirds of all existing Maintainers.
+
+Ideally, all project decisions are resolved by consensus. If impossible, any Maintainer may call a vote. Unless otherwise specified in this document, any vote will be decided by a [supermajority](#Supermajority) of Maintainers.
+
+In case of situation with not enough participation from maintainer for **non** critical decision we can lower the supermajority to [**simple majority**](#simple-majority).
+
+For any **critital** decisions [CNCF TOC](https://www.cncf.io/people/technical-oversight-committee/) should be consulted for appravals and moving forward.
+
+## Voting requirements:
+
+* Adding a Maintainer: [Supermajority](#Supermajority)
+
+* Removing a Maintainer:  [Supermajority](#Supermajority)
+
+* Requesting CNCF resources: [Simple majority](#simple-majority)
+
+* Charter changes: [Supermajority](#Supermajority)
```

Comment:
All these require super majority, right? Should we add that somewhere?

Reply:
Not sure what do you mean as the section is called: ## Voting requirements:
and Requesting CNCF resources: [Simple majority](#simple-majority) are simple majority

Reply:
I must be blind, I missed the link :facepalm: